server/pkg/schema/field_tag.go

```diff
@@ -0,0 +1,41 @@
+package schema
+
+var TypeTag Type = "tag"
+
+type FieldTag struct {
+	values       []string
+	defaultValue *string
```

Comment:
```suggestion
	defaultValue []string
```

Sorry I missed that it's tag so this allows multiple values.